application/lib/main.dart

```diff
@@ -150,7 +152,7 @@ class _MyHomePageState extends State<MyHomePage> {
               break;
             case 2:
               // tutorials
-              path = Routes.tutorialTwo;
+              path = Routes.tutorialSix;
```

Comment:
Think we'll want to redo the merge to use the more up-to-date navigation system where we just have the tutorial buttons on the home page, rather than in that bottom navigation bar. If we commit this, then we'll only be able to access tutorial 6 and not the others

Reply:
Just on this, I'll think we'll have to do `git merge --no-commit --no-ff` to ensure that we can manually merge into the updated `ApplicationPort` locally. That'll give us the most control over which bits to keep and which to change.